pesticide_backend/src/pesticide_app/permissions.py

```diff
@@ -1,6 +1,14 @@
 from rest_framework import permissions
 from pesticide_app.models import *
+import os
+import yaml
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_CONFIG_FILE = open(os.path.join(
+    BASE_DIR,
+    'config/base.yml'
+))
+BASE_CONFIGURATION = yaml.load(BASE_CONFIG_FILE, Loader=yaml.FullLoader)
```

Comment:
The config YAML file has already been loaded in settings.py file. Define the required variable in settings.py and then import the variable (like the flask token) from settings.py.

For example:

// in settings.py:
FLASK_TOKEN = BASE_CONFIGURATION["flaskToken"]["token"]

// in permissions.py:
from pesticide.settings import FLASK_TOKEN